feature.add-album-to-queue.md

```diff
@@ -0,0 +1,48 @@
+# Audio Player: Add Album to Queue
+**Authors:** [@szainmehdi](https://github.com/szainmehdi) [@shea786](https://github.com/shea786)  
+**Type:** Feature
+
+## Overview
+Allow the user to add the entire album to the audio player queue
+
+## Why are we making this change?
+So that it is easier for the user, instead of them going to each track to add to queue, one button would add the entire track
+
+## Requirements
+- User should be able to an entire album to the playlist
+- The order of which the tracks are being displayed is the order of which the tracks should be added to the queue
+- User should be able to add all tracks from the `Reciter Profile` page against each table to the queue
```

Comment:
I'm unsure what you mean here. Can you clarify?

Reply:
I guess I didnt explain it right, my bad

basically against each album on the reciter page, we want to show a button to play the all tracks from the album, re-worded in the new commit